tools/msvs/msi/nodemsi.wixproj

```diff
@@ -14,25 +14,25 @@
     <NodeVersion Condition=" '$(NodeVersion)' == '' ">0.0.0.0</NodeVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
-    <OutputPath>..\..\..\$(Configuration)\</OutputPath>
```

Comment:
What's the reason for removing $(Configuration) from the path? I don't know that we are going to release debug MSIs any time soon, but if it doesn't hurt to keep those paths separate then I think it might be preferable.